lib/aff4_objects/security.py

```diff
@@ -192,7 +193,9 @@ def CheckAccess(self, token):
       return True
 
     # Check that there are enough approvers.
-    lifetime = self.Get(self.Schema.LIFETIME)
+    lifetime = config_lib.CONFIG["ACL.token_expiry"] 
```

Comment:
If the default is defined in seconds and the changes are handled in the Schema it should be unnecessary to override the use of the schema or convert to seconds here.